pkg/yurthub/util/connrotation.go

```diff
@@ -79,11 +101,12 @@ func (d *Dialer) Close(address string) {
 	d.mu.Lock()
```

Comment:
Will the problem be resolved by expanding the lock scope in `Close` method to include the for loop, so that you don't need to copy the map. 

I don't see an immediate reason why the conn.Close() has to be called asynchronously.

Reply:
It may take a while to complete conn.Close(), and it may be conn.Dial() to create a new one immediately when the connection is closed. So in order not to block connection creation, choose to copy the map.